packages/atomic/src/components/commerce/result-template-components/atomic-product-link/atomic-product-link.tsx

```diff
@@ -0,0 +1,105 @@
+import {isUndefined} from '@coveo/bueno';
+import {InteractiveResult as InteractiveProduct} from '@coveo/headless/commerce';
+import {Product} from '@coveo/headless/commerce';
+import {Component, h, Prop, Element} from '@stencil/core';
+import {buildCustomEvent} from '../../../../utils/event-utils';
+import {
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../../utils/initialization-utils';
+import {getDefaultSlotFromHost} from '../../../../utils/slot-utils';
+import {getAttributesFromLinkSlot} from '../../../common/item-link/attributes-slot';
+import {LinkWithItemAnalytics} from '../../../common/item-link/item-link';
+import {Bindings} from '../../../search/atomic-search-interface/atomic-search-interface';
+import {
+  InteractiveProductContext,
+  ProductContext,
+} from '../product-template-decorators';
+
+/**
+ * The `atomic-product-link` component automatically transforms a search product title into a clickable link that points to the original item.
+ * @slot default - Lets you display alternative content inside the link
+ * @slot attributes - Lets you pass [attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes) down to the link element, overriding other attributes, to be used exclusively with an "a" tag such as `<a slot="attributes" target="_blank" download></a>`.
+ */
+@Component({
+  tag: 'atomic-product-link',
+  styleUrl: 'atomic-product-link.pcss',
+  shadow: false,
+})
+export class AtomicProductLink implements InitializableComponent {
```

Comment:
Do we want to prefix all product template components with 'product' ?

Reply:
I think so.
Or (even longer): `atomic-commerce-product-link`.

I think `atomic-product-link` is 👍